app/components/Header/index.js

```diff
@@ -110,7 +112,11 @@ class Header extends Component {
             >Exit</Button>
           </div>
         </div>
-        <ErrorMessage code={this.props.errorCode} />
+        {this.props.errorCode ?
```

Comment:
You should just be able to do, and I would prefer:

```
{this.props.errorCode ?  (<ErrorMessage code={this.props.errorCode} />) : this._renderUnsavedWarning()
```

I don't like mixing React create elements w/ jsx